Rulesets/level2.spectral.yaml

```diff
@@ -1,4 +1,4 @@
 extends:
   - spectral:oas
 rules:
-  no-$ref-siblings: warn # error is the default value
```

Comment:
Same applies here.

Reply:
fixed by adding the default ruleset as in apim-apps